package.json

```diff
@@ -141,5 +141,9 @@
     "turbo": "^1.11.3",
     "typescript": "^5.3.3",
     "webpack": "^5.89.0"
+  },
+  "volta": {
```

Comment:
Do you need to remove `.nvmrc` then?

Reply:
Nah, they can work independently in the same repo. Also I think NVM is more popular at CA.

Reply:
Should we even be committing this change then if no one else is meant to use it 😛

Reply:
I don't think there's any explicit standard that says we must use NVM... Just so happens to be the most common one found.